Cargo.toml

```diff
@@ -3,3 +3,6 @@ members=[
     'wasm-bindgen-derive',
     'tests'
 ]
+
+[profile.release]
+opt-level = 1
```

Comment:
Please add an explanation and a link to https://github.com/rustwasm/wasm-bindgen/issues/3457 here.

I am still not quite sure how to reproduce the bug properly; when compiling `wasm-bindgen-derive`, it only occurs for a release build, but in my test above it happens in both release and debug builds (and in the failed `nucypher-core` CI it also happened in debug, namely during a `clippy` run)